src/SSL.md

```diff
@@ -227,3 +227,55 @@ To assist with TLS/SSL issues, the following system property can be used:
 ```
 
 The property will provide a lot of logging output including the TLS/SSL handshake, that can be used to determine the cause of the problem. For more information please see [Debugging TSL/SSL Connections](http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/ReadDebug.html).
+
+### TLS/SSL for Hazelcast Management Center
+
+In order to use secured communication between cluster and Management Center, you have to configure the cluster. For Management Center configuration, please refer to [Using Management Center with TLS-SSL Only](http://docs.hazelcast.org/docs/management-center/3.9-EA2/manual/html/Using_Management_Center_with_TLS-SSL_Only.html).
+
+Please see the below example snippet to see the full declarative configuration at the member side:
```

Comment:
For the declarative part maybe we should refer to https://github.com/hazelcast/mancenter-reference-manual/commit/2ecd28f8e7538d47230a5ec9d52daae3ac207b12#diff-a45cbea69e98d0643eb3a78fd35e75aaR108 to avoid duplication.

Reply:
Oh, I wasn't aware of your latest commit. Instead, how about I added the programmatic configuration to your section and then linked it from other places? You know, to have programmatic and declarative configuration together? WDYT?

Reply:
Of course, that too makes sense.